_search-plugins/knn/knn-index.md

```diff
@@ -197,6 +231,6 @@ Setting | Default | Updateable | Description
 :--- | :--- | :--- | :---
 `index.knn` | false | false | Whether the index should build native library indices for the `knn_vector` fields. If set to false, the `knn_vector` fields will be stored in doc values, but Approximate k-NN search functionality will be disabled.
 `index.knn.algo_param.ef_search` | 512 | true | The size of the dynamic list used during k-NN searches. Higher values lead to more accurate but slower searches. Only available for *nmslib*.
-`index.knn.algo_param.ef_construction` | 512 | false | (Deprecated in 1.0.0. Use the mapping parameters to set this value instead.) Only available for *nmslib*. Refer to mapping definition.
+`index.knn.algo_param.ef_construction` | 512 | false | (Deprecated in 1.0.0. Use the mapping parameters to set this value instead.) Only available for nmslib. Refer to mapping definition.
```

Comment:
Are the parentheses necessary in these list items? It looks to me like we could safely remove them, which would avoid the weirdness of the ending punctuation inside them. Can we change to "Refer to the mapping definition." in all instances?

Reply:
Agreed. I replaced with:
"Deprecated in 1.0.0. Use the [mapping parameters](https://opensearch.org/docs/latest/search-plugins/knn/knn-index/#method-definitions) to set this value instead."
The mapping parameters will indicate what is appropriate for the different engines. So I think this will be adequate.